cmd/lncli/commands.go

```diff
@@ -2953,6 +2953,13 @@ var describeGraphCommand = cli.Command{
 				"graph. Unannounced channels are both private channels, and " +
 				"public channels that are not yet announced to the network.",
 		},
+		cli.BoolFlag{
+			Name: "calculate_node_centrality",
```

Comment:
How about just `--centrality`? As is, it's a bit long for a command line param IMO.

Reply:
done